src/pymor/tools/mpi.py

```diff
@@ -95,6 +96,17 @@ def event_loop_settings(auto_launch=True):
     return {'auto_launch': auto_launch}
 
 
+def launch_event_loop():
+    global _event_loop
+    if rank0:
+        from pymor.core import defaults
+        if defaults.defaults_changes() > 0:
+            call(defaults.set_defaults, defaults.get_defaults(_user=True, _file=False, _code=False))
```

Comment:
As above, I would say that `_file` should be `True` as well.

Reply:
Yes, you are right. This may break on a distributed memory system.